packages/@markuplint/parser-utils/src/get-location.ts

```diff
@@ -9,6 +9,13 @@ export function getCol(rawCodeFragment: string, startOffset: number) {
 	return (lines.at(-1) ?? '').length + 1;
 }
 
+export function getPosition(rawCodeFragment: string, startOffset: number) {
```

Comment:
@exoego Nice!

Please add the below JSDoc comment to getLine and getCol:
@deprecated Use {@link getPosition} instead. Will be removed in v5.0.0.

Reply:
Added deprecation https://github.com/markuplint/markuplint/pull/1484/commits/8b9b1b47634e70a10198688bcb6a1073e10be007